Blog - Ga4 vs posthog

Lior539 opened the pull request:
Blog post on Google Analytics vs PostHog.

Would like feedback on:
- I use "Google Analytics", "Google Analytics 4", and "GA4" interchangeably in the post. Let me know if it feels weird at any point
- Any major missing points?

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,497 @@
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-vs-kubit/posthog-vs-kubit.jpg
```
Will make art request

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,497 @@
+
+### 1. PostHog does more than analytics
+
+[Analytics](/product-analytics) is a core part of our platform, but it's just one component in a set of important tools for product teams. PostHog enables you to collect feedback with [surveys](/surveys), understand user behavior with [session replays](/session-replay), test changes with [A/B tests](/ab-testing), and deploy changes with [feature flags](/feature-flags).
```
The ./surveys page is not live yet - waiting on  https://github.com/PostHog/posthog.com/pull/6580

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,497 @@
+
+Compliance can be a bit of a regulatory headache, especially for international organizations who have to adhere to rules such as GDPR. PostHog makes such things simple by offering a choice of where your data is hosted, and a variety of security options to keep your data secure.
+
+In contrast, it's not clear whether GA4 is GDPR compliant. While Google provides an article on [GDPR privacy controls in GA4](https://support.google.com/analytics/answer/9019185), privacy watchdogs in some EU countries have [warned against using Google Analytics](https://techcrunch.com/2023/07/03/google-analytics-sweden-gdpr-fines/) for breaching GDPR rules.
```
Would love feedback on this paragraph: Im concerned that saying "it's not clear" comes across as not opinionated enough, but I don't want to outright say GA4 is illegal (Im not actually sure it is, and I also want to come across as fair)

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,497 @@
+
+GA4 has a free tier which should be sufficient for most startups and scale-ups. Larger enterprises with higher data collection and export requirements will need to upgrade to GA360, which starts at $50,000 per year. A good rule of thumb is that if you're exporting more than 1 million events per day, or querying data with more than 10 million events, you'll need to upgrade to GA360 (see a full bre ...
+
+PostHog has transparent pricing based on the usage. It’s free to get started and completely free for the first 1 million events. After this free monthly allowance you'll pay $0.00031/event, and PostHog charges progressively less the more you use. We recommend trying [our pricing calculator](/pricing) and reading our [guide for estimating your usage](/docs/billing/estimating-usage-costs).
```
```suggestion
PostHog has transparent pricing based on the usage. It’s free to get started and completely free for the first 1 million analytics events. After this free monthly allowance you'll pay $0.00031/event, and PostHog charges progressively less the more you use. We recommend trying [our pricing calculator](/pricing) and reading our [guide for estimating your usage](/docs/billing/estimating-usage-costs).
```

andyvan-ph reviewed (commented):
General feedback:

- Using GA4 / Google Analytics is fine. No worries there.

- Apologies for not mentioning this before, but Cory created a new table template that works better on mobile + looks better. It's only on [this Mixpanel alternatives article](https://posthog.com/blog/best-mixpanel-alternatives) atm. You can pull the template from that article to use here.

- It needs a bit more focus on why PostHog is great for engineers + web apps compared to GA. It's kind of there in tables, but we should make more of it. I already mentioned splitting up that long table, but maybe we should have a table comparing SDK support between them both?

- It feels a little too one-side atm. These comparisons should be feel like they're "in good faith". Off the top of my head, some things GA4 does not mentioned:
  - Natural language search for insights
  - Predictive / automated alerts to changes in metrics
  - Integration with Looker Studio dashboarding (it's actually quite good)
  
- We should have something here what kinds of companies should / do use each tool – e.g. GA4 is more popular among big enterprises, e-commerce websites etc. PostHog is better for B2B SaaS, startups etc.

- Ian added some screenshots to his [LaunchDarkly comparison](https://posthog.com/blog/posthog-vs-launchdarkly), which I think worked well. Worth adding one or two where you think they make sense / add something.

Lior539 commented on contents/blog/ga4-alternatives.md:
```diff
@@ -144,6 +144,8 @@ See [GDPR-compliant analytics tools](/blog/best-gdpr-compliant-analytics-tools)
 </table>
 </div>
 
+For a full comparison between PostHog and Google Analytics, check our [in-depth guide](/blog/posthog-vs-ga4).
```
```suggestion
Read our [PostHog and Google Analytics comparison](/blog/posthog-vs-ga4) for an in-depth look at the differences
```

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,497 @@
+            <td>Export events for use in Zaps.</td>
+        </tr>
+        <tr>
+            <td><strong>Stripe</strong></td>
```
removed

Lior539 commented:
New changes:
- Addressed comments to make it more balanced
- Addressed Andy's comments
- Made it clearer that GA4 integrates better with other Google products.

Also, here are Andy's comments which I have **not** included:

> GA4 is more popular among big enterprises

I haven't mentioned specifically that GA4 is more popular among big enterprises. I dont think they necessarily have a bigger advantage over us when it comes to large enterprises, except maybe that they integrate better with other Google products (which is called out). Let me know if Im wrong here?

> It needs a bit more focus on why PostHog is great for engineers + web apps compared to GA. It's kind of there in tables, but we should make more of it. I already mentioned splitting up that long table, but maybe we should have a table comparing SDK support between them both?

I've now called out the specific features now in the paragraphs before the table, and split the long table into two.

I decided to not compare SDK support specifically , since GA4 is accessible via API and there are a ton of open source repos for this (so I dont think we necessarily have an advantage)

> It feels a little too one-side atm. These comparisons should be feel like they're "in good faith". Off the top of my head, some things GA4 does not mentioned:
> * Natural language search for insights

It should feel more balanced now. I havent mentioned the natural language search feature though, since that goes under "ease of use/UX" which we don't really have a section for. Also, GA4 UX is pretty horrible otherwise, so I think best not mention this.

> * Ian added some screenshots to his [LaunchDarkly comparison](https://posthog.com/blog/posthog-vs-launchdarkly), which I think worked well. Worth adding one or two where you think they make sense / add something.

I have a look at those screenshots, but didnt feel they added much value. I considered some other screenshots but didnt think of anything that I thought would be good. I think the tables are sufficient.

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+GA4 has a free tier which should be sufficient for most startups and scale-ups. Larger enterprises with higher data collection and export requirements will need to upgrade to GA360, which starts at $50,000 per year. A good rule of thumb is that if you're exporting more than 1 million events per day, or querying data with more than 10 million events, you'll need to upgrade to GA360 (see a full bre ...
+
+PostHog has transparent pricing based on the usage. It’s free to get started and completely free for the first 1 million analytics events. After this free monthly allowance you'll pay $0.00031/event, and PostHog charges progressively less the more you use. We recommend trying [our pricing calculator](/pricing) and reading our [guide for estimating your usage](/docs/billing/estimating-usage-costs) ...
```
```suggestion
PostHog has transparent pricing based on the usage. It’s free to get started and for the first 1 million analytics events each month. After this free monthly allowance, you pay $0.00031/event, and PostHog charges progressively less the more you use. We recommend trying [our pricing calculator](/pricing) and reading our [guide for estimating your usage](/docs/billing/estimating-usage-costs).
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+### Does PostHog offer a free trial?
+
+With PostHog, you don’t need a free trial — it’s free to get started, with a generous monthly allowance of events, replays, and API calls. If you’re within this allowance, PostHog is free to use forever.
```
```suggestion
With PostHog, you don’t need a free trial — it’s free to get started, with a generous monthly allowance of events, replays, and feature flag requests. If you’re within this allowance, PostHog is free to use forever.
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Apps/Integrations</strong></p>
```
```suggestion
          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Apps and integrations</strong></p>
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+Want to understand how PostHog and Google Analytics 4 (GA4) are different? Here's the short answer:
+
+- **Google Analytics** is designed for marketers to understand user traffic, performance of different marketing channels, and return on ad spend.
```
Maybe mention e-commerce here? Seems like a big focus for them and not for us.

Lior539 replied:
I touch on this below

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Marketing analytics</strong></p>
```
```suggestion
          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Ad analytics</strong></p>
```
I would argue we do have marketing analytics, but this is a good point.

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Real-time reporting</strong></p>
```
I will make the argument in favor, our real-time reporting is fairly similar to theirs, we just don't make any promises about it: https://posthog.com/templates/real-time-dashboard

Lior539 replied:
Oh nice, didnt realize we had this. GA4 leans heavily into this, and we dont. So I dont think we should promise that we do. Will leave as it

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+  </table>
+</div>
+
+## Analytics features comparison
```
I think a big one GA does a good job of that we don't (that is missing) is monetization. In-app purchases, purchase conversion, etc. When data warehouse is complete, maybe we revisit this:
 
![Screen Shot 2023-10-16 at 1 17 58 PM](https://github.com/PostHog/posthog.com/assets/34755028/e1dcbee4-3387-46be-8354-022137a4b6ee)

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Lifecycle insights</strong></p>
```
They do have new and returning, but maybe this is being a bit generous to them:

https://support.google.com/analytics/answer/11004084?sjid=10429453555889214577-NA#zippy=%2Cnew-users-and-returning-users

Lior539 replied:
Nice. Thanks for this, didnt realize they had it

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>SQL access</strong></p>
```
Maybe a note that GA lets you query with SQL if you export to BigQuery, but in PostHog it is built in?

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+## Apps and integrations
+
+Integrations are another major point of difference between GA4 and PosthHog, especially when it comes to importing data. Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
```
```suggestion
Integrations are another major point of difference between GA4 and PostHog, especially when it comes to importing data. Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
```
+ examples (API, SDKs, snippet for no-code sites)?

ivanagas replied:
https://posthog.com/tutorials/webflow and https://posthog.com/tutorials/framer-analytics are useful for showing how easy it is.

Lior539 replied:
> examples (API, SDKs, snippet for no-code sites)?

It's easy to log events with GA4 from no-code sites, sdks etc.
Importing data here refers to bringing in data from other sources that aren't your app

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+On the other hand, it's hard to import data into GA4 because:
+
+1. The type of data you're allowed to [import into GA4](https://support.google.com/analytics/answer/10071301) is limited. 
```
```suggestion
1. The data type and format you're allowed to [import into GA4](https://support.google.com/analytics/answer/10071301) is restrictive. 
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+On the other hand, it's hard to import data into GA4 because:
+
+1. The type of data you're allowed to [import into GA4](https://support.google.com/analytics/answer/10071301) is limited. 
+2. You need to constantly upload CSV files manually, or you need to set up an SFTP server to automatically do this for you.
```
```suggestion
2. You either need to constantly upload CSV files manually or set up an SFTP server to automatically do this for you.
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+    </thead>
+    <tbody>
+      <tr>
+        <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Zapier</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Export events for use in Zaps.</p></td>
```
Is this the number one Zapier use case?

"Trigger Zaps with events" instead maybe?

Lior539 replied:
```suggestion
        <td><p class="!mb-0 pb-0.5 !leading-tight"><strong>Zapier</strong></p><p class="!mb-0 !text-sm text-opacity-75 leading-none">Trigger Zaps with events.</p></td>
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+## Privacy, compliance, and security
+
+Compliance can be a bit of a regulatory headache, especially for international organizations who have to adhere to rules such as GDPR. PostHog makes such things simple by offering a choice of where your data is hosted, cookie-less tracking, and a variety of security options to keep your data secure. 
```
```suggestion
Compliance can be a bit of a regulatory headache, especially for international organizations who must adhere to rules such as GDPR. PostHog makes such things simple by offering a choice of where your data is hosted, cookie-less tracking, and a variety of security options to keep your data secure. 
```

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+> Familiar with Google Analytics and want more information on how to use PostHog? We've written a [guide to PostHog for Google Analytics users](/blog/google-analytics-to-posthog).
+
+### How much do PostHog and Google Analytics cost?
```
Screenshots would be nice in this section

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+
+<ArrayCTA /> 
+
+## PostHog and Google Analytics feature comparison
```
1. Autocapture? I've heard complaints about how long it takes to implement GA (and how expensive it is). Especially GA4 makes this complicated. 
2. Mobile analytics. Both have it, but focus on different areas. Potentially worth noting.

ivanagas replied:
On autocapture, looks like GA4 has it under "enhanced analytics": https://support.google.com/analytics/answer/9216061

Lior539 replied:
> Autocapture? I've heard complaints about how long it takes to implement GA (and how expensive it is). Especially GA4 makes this complicated.

Hmm this wasn't my experience with it when I set it up a few days ago. It was just pasting a few lines of code and it began auto-capturing events

> Mobile analytics. Both have it, but focus on different areas. Potentially worth noting.

Hmm mind elaborating? Seems like both PostHog and GA4 do similar things (event capture and user properties/demographics)

https://firebase.google.com/docs/analytics/get-started?technology=ios&platform=ios
https://firebase.google.com/docs/analytics/get-started?technology=android&platform=android

andyvan-ph replied:
Love Google's name for things. Enhanced analytics is the vaguest, most meaningless term.

ivanagas commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,536 @@
+    </table>
+</div>
+
+## Frequently asked questions 
```
Comparison of how long it takes to implement each? PostHog is relatively easy setup IMO.

ivanagas reviewed (approved):
Looks solid, realize this is a tough one because its hard to pin down what GA's features actually are. Left some ideas/points to add.

Since there is a lot of information to share, I think visuals could help a lot. 
1. Pricing comparisons
2. Integration lists
3. Example dashboards

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+Want to understand how PostHog and Google Analytics 4 (GA4) are different? Here's the short answer:
+
+- **Google Analytics** is designed for marketers to understand user traffic, performance of different marketing channels, and return on ad spend.
```
```suggestion
- **Google Analytics** is primarily designed for marketers who want to analyze landing pages, compare marketing channels, and return on ad spend.
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+- **Google Analytics** is designed for marketers to understand user traffic, performance of different marketing channels, and return on ad spend.
+
+- **PostHog** is an all-in-one platform built to help engineers create better products. It includes analytics and a whole bunch more – like session replays, feature flags, A/B testing, and surveys.
```
```suggestion
- **PostHog** is an all-in-one platform built to help engineers understand users and create better products. It includes analytics and a whole bunch more, such as session replays, feature flags, A/B testing, and surveys.
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+## PostHog and Google Analytics feature comparison
+
+As mentioned above, PostHog offers a wider selection of features than GA4, which focuses solely on analytics. This makes PostHog better suited for teams who are building products.
```
```suggestion
PostHog offers a wider selection of features than GA4, which focuses solely on analytics. This makes PostHog better suited for teams who are building products.
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+  </table>
+</div>
+
+## Analytics features comparison
```
```suggestion
## Analytics features
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+<ArrayCTA /> 
+
+## PostHog and Google Analytics feature comparison
```
```suggestion
## Core features
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Marketing analytics</strong></p>
```
I think this implies you can't do marketing analytics using PostHog, which isn't true. Maybe "Advertising analytics" would be better?

Lior539 replied:
haha exactly what Ian said

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+## Privacy, compliance, and security
+
+Compliance can be a bit of a regulatory headache, especially for international organizations who must adhere to rules such as GDPR. PostHog makes such things simple by offering a choice of where your data is hosted, cookie-less tracking, and a variety of security options to keep your data secure. 
```
```suggestion
PostHog makes GDPR compliance easy by letting you choose where your data is hosted: EU or US. PostHog also offers cookie-less tracking, and a variety of security options to keep your data secure. 
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+Compliance can be a bit of a regulatory headache, especially for international organizations who must adhere to rules such as GDPR. PostHog makes such things simple by offering a choice of where your data is hosted, cookie-less tracking, and a variety of security options to keep your data secure. 
+
+GA4 also offers various [privacy controls](https://support.google.com/analytics/answer/9019185), but does not enable you to choose where your data is stored.
```
```suggestion
GA4 also offers various [privacy controls](https://support.google.com/analytics/answer/9019185), but you can't choose where your data is stored.
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+## Apps and integrations
+
+Integrations are another major point of difference between GA4 and PosthHog, especially when it comes to importing data. Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
```
```suggestion
Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+
+Integrations are another major point of difference between GA4 and PosthHog, especially when it comes to importing data. Because PostHog is built to be your single source of truth for viewing data, it's very simple to import data from other sources. 
+
+On the other hand, it's hard to import data into GA4 because:
```
```suggestion
In contrast, it's hard to import data into GA4 because:
```

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+### 3. PostHog is open source and transparent
+
+PostHog is built with transparency at its core. We work in the open and give full access to our [source code](https://github.com/PostHog), and enable you to [build integrations](/docs/apps/build) and [other services](/blog/how-we-built-an-app-server) on top of the product. You can even give feedback on [our public roadmap](/roadmap).
+
+<ArrayCTA /> 
```
```suggestion
```

A bit too early (I know some older ones have a CTA here!)

andyvan-ph commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+    </tbody>
+  </table>
+</div>
+
```
```suggestion

<ArrayCTA /> 
```

andyvan-ph replied:
I think this is a better place for the first CTA

andyvan-ph reviewed (approved):
Left some small line edits, but this looks good to me.

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+---
+date: 2023-10-12
```
```suggestion
date: 2023-10-17
```

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,546 @@
+      </tr>
+      <tr>
+        <td>
+          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Marketing analytics</strong></p>
```
```suggestion
          <p class="!mb-0 pb-0.5 !leading-tight"><strong>Advertising analytics</strong></p>
```

Lior539 commented on contents/blog/posthog-vs-ga4.md:
```diff
@@ -0,0 +1,545 @@
+  </table>
+</div>
+
+<ArrayCTA /> 
+## Analytics features
```
```suggestion
<ArrayCTA /> 

## Analytics features
```
